e2e-pw/src/oss/poms/saved-views.ts

```diff
@@ -0,0 +1,406 @@
+import { expect, Locator, Page } from "src/oss/fixtures";
+
+const ColorList = [
+  "Gray",
+  "Blue",
+  "Purple",
+  "Red",
+  "Yellow",
+  "Green",
+  "Pink",
+  "Orange",
+  "Purple",
+];
+type Color =
+  | "Gray"
+  | "Blue"
+  | "Purple"
+  | "Red"
+  | "Yellow"
+  | "Green"
+  | "Pink"
+  | "Orange"
+  | "Purple";
+
+const updatedView = {
+  name: "test updated",
+  description: "test updated",
+  color: "Yellow" as Color,
+};
+
+export const updatedView2 = {
+  name: "test updated 2",
+  description: "test updated 2",
+  color: "Orange" as Color,
+};
```

Comment:
these fixtures have nothing to do with the "model" of a saved view and therefore should live in the spec file.